secure_message/services/party_service.py

```diff
@@ -17,13 +17,14 @@ class PartyService:
     @staticmethod
     def get_url(api_param, code):
         """
-        :param api_param:  is the key configuration that that represent part or the URI .
+        :param api_param:  is the key configuration that represent part or the URI .
```

Comment:
Should this read "is the key configuration that represent**s** part **of** the URI"?

Extra space between "URI" and "."